bridge_adaptivity/module/models.py

```diff
@@ -19,13 +20,29 @@
 log = logging.getLogger(__name__)
 
 
+class ModelWithDefaultInstanceMixin(object):
+    IS_DEFAULT_FIELD = 'is_default'
+
+    def save(self, *args, **kwargs):
+        if not hasattr(self, self.IS_DEFAULT_FIELD):
+            raise ValueError('Model {} has no field {}'.format(self, self.IS_DEFAULT_FIELD))
+
+        if getattr(self, self.IS_DEFAULT_FIELD, None):
+            default_qs = self.__class__.objects.filter(**{self.IS_DEFAULT_FIELD: True})
+            if default_qs:
+                default_qs.update(is_default=False)
+        return super(ModelWithDefaultInstanceMixin, self).save(*args, **kwargs)
+
+
 @python_2_unicode_compatible
 class Sequence(models.Model):
     """Represents User's problem solving track."""
 
     lti_user = models.ForeignKey(LtiUser)
     collection = models.ForeignKey('Collection')
     engine = models.ForeignKey('Engine', blank=True, null=True)
+    collection_group = models.ForeignKey('CollectionGroup', blank=True, null=True)
```

Comment:
not needed, sequence is interested in the engine and grading policy, not in the collection_group